### PR TITLE
Fixes archer-search-records bug

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/45630)

## Description
Fixes the Archer search record bug when searching with incident id.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py [hunk 1]
```diff
@@ -470,22 +478,25 @@ def search_records(
         search_field_name = ''
         search_field_id = ''
         fields_mapping = level_data['mapping']
+        level_id = level_data['level']
```
Comment (dantavori) [inline Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py:481]: can you elaborate on this field? will it always appear in the level data?

**Review by dantavori — APPROVED** [review]
looks great

File: Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py (hunk 1 above)
Comment (yaakovpraisler) [reply Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py:481]: We add it in the code [here](https://github.com/demisto/content/blob/master/Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py#L354)

**Comment by yaakovpraisler** [comment]
Validation fails due to non-updated docker image. It was not updated as it fails the build.
@Itay4 - Need a force merge.

**Pull request merged**